### ISSUE TITLE
Feature request: Allow text and labels to be resized

### ISSUE THREAD

**Issue by ChildishGiant** [brain-child/moobo#11]

**Comment by pierrefabarius** [comment]
If you mean increasing/decreasing font-size, you can do that with ctrl/+ and ctrl/-.

**Comment by ChildishGiant** [comment]
![image](https://user-images.githubusercontent.com/13716824/141007125-ded4b650-397e-446a-b329-3eb9e1f586e8.png)
That should probably be bound to the scale item in the menu, so you don't need to know the shortcut or even use the keyboard

**Comment by pierrefabarius** [comment]
Yeah, that totally makes sense! Feel free to implement this feature.
